lib/internal/test_runner/utils.js

```diff
@@ -290,6 +302,9 @@ function parseCommandLine() {
     coverageIncludeGlobs,
     destinations,
     forceExit,
+    minimumBranchCoveragePercent,
+    minimumFunctionCoveragePercent,
+    minimumLineCoveragePercent,
```

Comment:
Even though `run()` does not support coverage yet, I think you should give these names that we would be happy having as part of the public `run()` API. We can change these later since right now they are only internal, but I'd rather not have to rename them later.

What do you think about these names for the flags and options:

- `--test-coverage-branches` and `branchCoverage`
- `--test-coverage-functions` and `functionCoverage`
- `--test-coverage-lines` and `lineCoverage`

The `min-` part of the CLI flags feels a bit unnecessary, and I dont' think the option names should include the word "percent" (if anything, "threshold" would be more appropriate).

Reply:
I've updated the names to `--test-coverage-...-threshold`, WDYT?

Reply:
This is just my opinion, but it seems a bit wordy to be honest.

For the flags, `--test` indicates that the flag is related to the test runner, and `coverage` indicates that the flag is related to code coverage. I'm not sure the `threshold` part gets us a lot. For comparison, c8, which doesn't need to worry about any namespacing calls these `--lines`, `--functions`, etc.

The same thing goes for the `run()` options.

Reply:
In my opinion, telling users that this is a threshold flag/option is important, but I'm curious to see what others have to say.

Reply:
I concur with @cjihrig

Reply:
Done :-)